src/grid_overlay.cpp

```diff
@@ -113,7 +122,7 @@ void addKnownFreePoint (OverlayClouds* overlay, const gm::Point& p, const double
       {
         const index_t ind = cellIndex(geom, c2);
         overlay->hit_counts[ind] = 0;
-        overlay->pass_through_counts[ind] = overlay->min_pass_through+1;
+        overlay->pass_through_counts[ind] = overlay->min_pass_through * increment(overlay.occupancy_threshold) + 1;
```

Comment:
Just check all places where `pass_through_counts` is used and updated/scaled the `min_pass_through` value.